amaascore/assets/foreign_exchange.py

```diff
@@ -65,29 +48,69 @@ def country_codes(self, country_codes):
 
 class ForeignExchange(ForeignExchangeBase):
```

Comment:
I might be wrong, but seems this class 'ForeignExchange' is now not used after we have separate classes for spot, forward and option. Should we still keep this class if that's the case?

Reply:
This will be the *actual* asset on things like the asset screens, lookups, etc.  So e.g. if a currency pair was missing, the user could create their own ForeignExchange class, which could then be traded (which would instantiate the relevant ForeignExchangeSpot class).

Reply:
I assume their self created ForeignExchange asset will be stored under the "underlying" attribute when they instantiate a ForeignExchangeSpot or ForeignExchangeForward class right?

Reply:
You got it!